src/pseudopeople/interface.py

```diff
@@ -290,10 +293,13 @@ def generate_american_community_survey(
 
     :param state:
 
-        The state string to include in the dataset. Either full name or
-        abbreviation (e.g., "Ohio" or "OH"). Will raise a ValueError if
-        there is no data for this state. If None is provided, data from
-        all locations are included in the dataset.
+        The US state for which to generate simulated American Community
+        Surveys of the simulated population, or `None` (default) to
+        generate data for all available US states. The returned dataset
+        will contain survey data for simulants living in the specified
+        state during the specified year. Can be a full state name or a
+        state abbreviation (e.g., "Ohio" or "OH"). Will raise a
+        `ValueError` if there is no data for this state.
```

Comment:
Seems like we should put `ValueError` in the `raises` section along with `ConfigurationError` and `DataSourceError`. If we do that, should we leave this sentence here as well, or should we get rid of it? The same comment and question applies to the `ValueError` raised for an invalid `year`.

Reply:
Agreed that it should be in `raises`. In `raises` it should say both possible causes: invalid year or invalid state. I don't have a strong feeling one way or the other about whether we should remove this sentence from the parameter description.